source/features/separate-draft-pr-button.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const previewForm = select('.new-pr-form')! as HTMLFormElement;
+
+	// Move the "Allow edits from maintainers" to its own line if it exists
+	const allowEditsCheckbox = select('.float-left', previewForm);
+	if (allowEditsCheckbox) {
+		allowEditsCheckbox.classList.remove('float-left');
+	}
```

Comment:
If you shorten the label to just “create draft” you probably won’t have to mess with this checkbox at all.

Reply:
True, but I think it might then be confusing for users not familiar with draft pull requests that they actually *are* pull requests:

![Unbenannt](https://user-images.githubusercontent.com/202916/66700377-ce0fed80-ecef-11e9-91e0-64629f68b3cb.png)

Can we assume that all users of Refined GitHub already know how draft PRs work?

Also, we then deviate from the official GitHub wording (meaning that we also have to implement a special case in the button adding code below).

Reply:
The H1 on the page is “Open a pull request” so it would contextually make sense. 

If we want to keep the name in the button it could be a shorter “Create draft PR”